client/src/Components/ItemsForms.jsx

```diff
@@ -0,0 +1,40 @@
+export function ItemForms({ numberOfItems, out }) {
```

Comment:
This confused me. I thought it was going to return multiple `<form>` elements. Instead, it looks like it's returning multiple items that you will put into a single form. "FormItems" would be a more clear name.